curriculum/challenges/english/20-upcoming-python/learn-recursion-by-solving-the-tower-of-hanoi-puzzle/64de6c8a5305d8173a3a9e09.md

```diff
@@ -0,0 +1,70 @@
+---
+id: 64de6c8a5305d8173a3a9e09
+title: Step 32
+challengeType: 20
+dashedName: step-32
+---
+
+# --description--
+
+`make_allowed_move` takes in `rod1` and `rod2` as parameters. You need a little refactoring here. Change each occurence of `source` into `rod1`.
```

Comment:
```suggestion
`make_allowed_move()` takes in `rod1` and `rod2` as parameters. You need a little refactoring here. Change every occurrence of `source` into `rod1`.
```

Reply:
Was the term "refactoring" explained in a previous project? I would suggest adding more context about this in this step.